src/sentry/api/endpoints/organization_events.py

```diff
@@ -78,6 +80,16 @@ def get_v2(self, request, organization):
         try:
             params = self.get_filter_params(request, organization)
             snuba_args = self.get_snuba_query_args_v2(request, organization, params)
+
+            fields = snuba_args.get('selected_columns')
+            groupby = snuba_args.get('groupby', [])
+
+            if not fields and not groupby:
+                return Response({'detail': 'No fields or groupings provided'}, status=400)
+
+            if any(field for field in groupby if field not in ALLOWED_GROUPINGS):
+                return Response({'detail': 'Invalid groupby value requested'}, status=400)
```

Comment:
Should we indicate what the valid options are?

Reply:
Sounds good